exporter/awsxrayexporter/internal/translator/http.go

```diff
@@ -111,6 +109,12 @@ func makeHTTP(span ptrace.Span) (map[string]pcommon.Value, *awsxray.HTTPData) {
 		return true
 	})
 
+	// if there is no network peer address, then `client.address` is from X-Forwarded-For.
```

Comment:
I don't really understand this. Can you. explain? I'm trying to follow using the migration guide but maybe there is information elsewhere?